### DISCUSSION TITLE
[Feature] Be able to delete Video/Photo in album view

### DISCUSSION THREAD

**Discussion by danielyeriksson** [immich-app/immich#2579] [Feature Request]
### The feature

Would be nice to be able to clean up and delete files via the albums instead of having to check the date in the album and then hope that a lot of the images in the album are around that date.

So under the three dots, or maybe have an option when you press delete to "Delete from permanently" or something like that

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by tsemachh** [comment]
Would add - when deleting albums allow also deleting the assets in the album

**Comment by jrasm91** [comment]
This has been implemented for a while now.

![Screenshot_20231024-151901.png](https://github.com/immich-app/immich/assets/4334196/d5523382-e3d4-4c43-91e2-1213dcf0c62e)
